app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -138,11 +138,13 @@ Template.visitorInfo.helpers({
 	roomOpen() {
 		const room = ChatRoom.findOne({ _id: this.rid });
 
-		return room.open;
+		return room.open && (hasRole(Meteor.userId(), 'livechat-manager') || (room.servedBy && room.servedBy._id === Meteor.userId()));
 	},
 
-	guestPool() {
-		return settings.get('Livechat_Routing_Method') === 'Guest_Pool';
+	canReturnQueue() {
+		const config = Template.instance().routingConfig.get();
+		const { returnQueue } = config;
+		return returnQueue;
```

Comment:
Why not `return config.returnQueue`? And is it possible to config be null?

Reply:
The `config` will always be an object.
Done.